test/kubemark/hollow-kubelet_template.json

```diff
@@ -0,0 +1,47 @@
+{
+	"kind": "ReplicationController",
+	"apiVersion": "v1",
+	"metadata": {
+		"name": "hollow-kubelet",
+		"labels": {
+			"name": "hollow-kubelet"
+		}
+	},
+	"spec": {
+		"replicas": ##numreplicas##,
```

Comment:
It's for the sed command in the 'start-kubemark.sh' script. I don't think that the downward API can pass an arbitrary variable, so I needed another way to do this.